src/components/attendee/ScheduleOverlay/ScheduleOverlay.tsx

```diff
@@ -338,32 +60,31 @@ export const ScheduleOverlay: React.FC = () => {
         onChange={togglePersonalisedSchedule}
         title="Only show bookmarked events"
       />
-      <div className={CN.scheduleDaysWrapper}>
-        <div className={CN.scheduleDays}>{weekdays}</div>
-        {weekdays.length > minWeekDaysScrollValue && (
-          <div className={CN.scheduleDaysArrows}>
-            <FontAwesomeIcon
-              icon={angleLeft}
-              className={backAngleClasses}
-              onClick={handleWeekScrollBackward}
-            />
-            <FontAwesomeIcon
-              icon={angleRight}
-              className={forwardAngleClassnames}
-              onClick={handleWeekScrollForward}
-            />
-          </div>
-        )}
-      </div>
+      {dayDifference <= minWeekDaysScrollValue ? (
+        <Weekdays
+          setSelectedDayIndex={setSelectedDayIndex}
+          selectedDayIndex={selectedDayIndex}
+          isScheduleTimeshifted={isScheduleTimeshifted}
+        />
+      ) : (
+        <ScrollableWeekdays
+          setSelectedDayIndex={setSelectedDayIndex}
+          selectedDayIndex={selectedDayIndex}
+          dayDifference={dayDifference}
+          isScheduleTimeshifted={isScheduleTimeshifted}
```

Comment:
[code smell] These names are contrary to common HTML and React practices. Handlers usually have name of `onSomething` and it's easy to make the parent take the arguments from the handler and call it's own `setWhatever` function.

[OOS] probably
The components as they are now look like tightly coupled almost as if having extra component adds more noise in passing values and listeners than helping compartmentalize logic.

There should be a way to improve the code, but I'm afraid it may take too much time and be OOS for this PR.

Reply:
Replaced with `onSelectDayIndex`, maybe will decouple them in the future [ede5a78](https://github.com/sparkletown/sparkle/pull/2921/commits/ede5a78fee222b03794b50d81d1b5bb452d05814)